dev/validate_bom__main.py

```diff
@@ -163,6 +165,7 @@ def main(options, metrics):
       deployer.collect_logs()
     if options.deploy_undeploy:
       deployer.undeploy()
+      validate_bom__config.teardown_environments(options)
```

Comment:
should there be a try/finally so the teardown happens even if undeploy fails? I'm not sure, but suspect it should. Manually undeploying isnt so bad but manually tearing down might be, so better to make a stronger effort.

Reply:
Done.